examples/adb-exfiltration-protection/README.md

```diff
@@ -24,7 +24,7 @@ Resources to be created:
 
 1. Update `terraform.tfvars` file and provide values to each defined variable
 2. (Optional) Configure your [remote backend](https://developer.hashicorp.com/terraform/language/settings/backends/azurerm)
-4. Run `terraform init` to initialize terraform and get provider ready.
+3. Run `terraform init` to initialize terraform and get provider ready.
```

Comment:
really, it's just enough to specify `1.` for each of the lines, and it will be numbered automatically. https://www.markdownguide.org/basic-syntax/#ordered-lists